src/components/ContentSidebar/ActivityFeed/activity-feed/ActivityFeed.js

```diff
@@ -162,28 +174,47 @@ class ActivityFeed extends React.Component<Props, State> {
         this.setState({ mentionSelectorContacts: getMentionWithQuery(searchStr) });
     };
 
+    componentDidMount(): void {
+        const { comments, tasks, versions } = this.props;
+        this.sortFeedItems(comments, tasks, versions);
+    }
+
+    componentWillReceiveProps(nextProps: any): void {
+        const { comments, tasks, versions } = nextProps;
+        this.sortFeedItems(comments, tasks, versions);
+    }
+
+    /**
+     * Sort feed valid feed items with new at the top, old at the bottom.
+     *
+     * @param args - Arguments list of each item container type that is allowed in the feed.
```

Comment:
We've been adding the param type up here, unnecessary with flow, but consistency.